README.md

```diff
@@ -3,14 +3,14 @@
 Here I will describe my journey in learning the Ruby programming language.
 The book I use is "Programming Ruby" (The Pragmatic Programmer's Guide).
 This repository is regularly updated with personal notes and some
-programs as well.
+example code and programs as well.
 
 
 ## Ruby as an object oriented language
 
 * Unlike other languages, everything in Ruby is an object, including
   return values of methods
-* Class is a comnination of state (variables) and functions that use
+* Class is a combnination of state (variables) and functions that use
```

Comment:
Corrected in https://github.com/renderedtext/ruby-101/tree/rexich/cmdline-args :+1: